scripts/change_url

```diff
@@ -1,29 +1,29 @@
 #!/bin/bash
 
-#=================================================
-# GENERIC STARTING
-#=================================================
-# IMPORT GENERIC HELPERS
-#=================================================
-
 source _common.sh
 source /usr/share/yunohost/helpers
 
 #=================================================
-# STANDARD MODIFICATIONS
+# INSTALL DEPENDENCIES
+#=================================================
+ynh_script_progression "Installing NodeJS..."
+
+# Install Nodejs
+ynh_nodejs_install
```

Comment:
Hmmm I'm not sure you really need to reinstall nodejs when changing the app url ?

Reply:
I had to, because chang_url failed without it